src/citra_qt/multiplayer/join_room_dialog.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <QPushButton>
+#include <iostream>
+
+#include "common/logging/log.h"
+#include "net_core/net_core.h"
+#include "citra_qt/multiplayer/join_room_dialog.h"
+#include "ui_join_room.h"
+
+JoinRoomDialog::JoinRoomDialog(QWidget* parent) : QDialog(parent), ui(new Ui::JoinRoomDialog) {
+    ui->setupUi(this);
+    RoomMember* room_member = NetCore::getRoomMember();
+    if (room_member->IsConnected()) {
+        ui->edit_nickname->setEnabled(false);
+        ui->edit_room_ip->setEnabled(false);
+        ui->spin_port->setEnabled(false);
+        ui->buttonBox->button(QDialogButtonBox::Ok)->setText("Leave");
```

Comment:
if you create widgets through the ui file you should also set the labels there.
In general, please wrap translatable strings in `tr("Foo")`